DopplerJobTest/DopplerSapServiceTests.cs

```diff
@@ -28,7 +28,8 @@ public DopplerSapServiceTests()
             _dopplerSapServiceSettingsMock.Setup(x => x.CurrentValue)
                 .Returns(new DopplerSapConfiguration
                 {
-                    Url = "https://localhost:5001/SetCurrencyRate"
+                    CurrencyEndpoint = "https://localhost:5001/SetCurrencyRate",
+                    BillingEndpoint = "https://localhost:5001/Billing/CreateBillingRequest"
```

Comment:
Minor change. In the local environment we are using the 65100 port.
```suggestion
                    CurrencyEndpoint = "https://localhost:65100/SetCurrencyRate",
                    BillingEndpoint = "https://localhost:65100/Billing/CreateBillingRequest"
```